lib/Auth/index.js

```diff
@@ -194,6 +196,10 @@ class Auth {
         begin = new Date().getTime();
         ({ status, body } = await superagent(method, url)
           .set(opts.headers)
+          .timeout({
+            response: opts.timeout || 60000,
```

Comment:
Let's default `opts.timeout` and `opts.deadline` here: https://github.com/myDevicesIoT/cayenne-core-js/blob/a97254b83312d97b4b10a1de878d137a38e1f513/lib/Auth/index.js#L163

```js
opts = Object.assign({ headers: {}, isPublic: false, cache: false, timeout: 60000, deadline: 90000 }, opts);
```

Reply:
added default value to opts.